hub: add archive repo (#21553)
docker/docs: content/manuals/docker-hub/repos/archive.md

---
description: Learn how to archive or activate a repository on Docker Hub
keywords: Docker Hub, Hub, repositories, archive, activate
title: Archive or activate a repository
linkTitle: Archive
toc_max: 3
weight: 35
---

You can archive a repository to avoid using outdated or unsupported images in

> your workflow. Docker Hub helps you identify outdated repositories that haven't
> been updated in over a year by displaying an icon ({{< inline-image
> src="./images/outdated-icon.webp" alt="outdated icon" >}}) next to the
> repositories on the [**Repositories** page](https://hub.docker.com/repositories/).

Reviewer: ```suggestion
your workflow. Outdated repositories are defined as repositories that haven't
been updated in over a year. Docker Hub displays an icon ({{< inline-image src="./images/outdated-icon.webp" alt="outdated icon" >}}) next to outdated repositories on the [**Repositories** page](https://hub.docker.com/repositories/).
```

Reviewer: just a little reword for clarity

Author: Yea, agree. That sentence is gnarly. 
For the suggested change, we don't define what outdated repositories are. I just realized that I implied that though. They may be over a year old, but still relevant and not outdated.

How about something like:
`Docker Hub identifies repositories that haven't been updated in over a year. These repositories are identified with an icon ( X ) on the Repositories page.`
